testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/messages/messages_en.properties

```diff
@@ -0,0 +1,4 @@
+doAccept=Yes
```

Comment:
I've already merged. I wouldn't rename the test as this is testing the terms and conditions feature. I'd rather add a separate CustomThemeTest.